examples/builder.rs

```diff
@@ -11,7 +11,9 @@ fn build_cli() -> Command<'static> {
                         .value_name("SHELL")
                         .help("The shell to generate the completions for")
                         .required(true)
-                        .possible_values(clap_complete_command::Shell::possible_values()),
+                        .value_parser(clap::builder::PossibleValuesParser::new(
+                            clap_complete_command::Shell::possible_values(),
+                        )),
```

Comment:
It would be nice to be able to reduce this boilerplate since this feels awkward. It also looks like `EnumValueParser` would be better than `PossibleValuesParser`.

Reply:
Done, force-pushed change